package.json

```diff
@@ -2,10 +2,15 @@
   "name": "react-flexview",
   "version": "1.0.1",
   "description": "A react component to abstract over flexbox",
-  "main": "index.js",
+  "files": [
+    "lib",
+    "react-flexview.d.ts"
+  ],
+  "main": "lib/FlexView.js",
```

Comment:
NOTE: if we re-add `src/index.js` (see last comment) this should become `"main": "lib",`